lib/android/app_methods.dart

```diff
@@ -52,6 +52,10 @@ Future<int> getAndroidSdkVersion() async {
   return await appMethodsChannel.invokeMethod('getAndroidSdkVersion');
 }
 
+Future<int?> getPrimaryColor() async {
+  return await appMethodsChannel.invokeMethod('getPrimaryColor');
+}
```

Comment:
Might be nicer if this returns a Future<Color?> instead, by moving the conversion here.

Reply:
fixed